ibis/backends/pyspark/__init__.py

```diff
@@ -11,7 +11,14 @@
 from packaging.version import parse as vparse
 from pyspark import SparkConf
 from pyspark.sql import SparkSession
-from pyspark.sql.functions import PandasUDFType, pandas_udf
+from pyspark.sql.functions import (
+    PandasUDFType,
+    col,
+    from_json,
+    pandas_udf,
+    struct,
+    to_json,
+)
```

Comment:
The pattern we use for this module is `import pyspark.sql.functions as F` and then refer to the module's objects as `F` wherever they are used. Having a `col` thing floating around IMO makes this code hard to understand given how many things something named `col` could refer to.